libs/gi/page-team/src/CharacterDisplay/Content.tsx

```diff
@@ -146,44 +147,58 @@ function TabNav({
       }}
     >
       {!hideTitle && (
-        <CardContent
-          sx={{
-            display: 'flex',
-            justifyContent: 'center',
-            p: 0,
-            textShadow: '#000 0 0 10px !important',
-            position: 'relative',
-          }}
-        >
-          <CardActionArea onClick={() => setEditMode(true)} sx={{ p: 2 }}>
+        <>
+          <CardActionArea onClick={() => setEditMode(true)}>
             <BootstrapTooltip
```

Comment:
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/b3ee67b3-2dcb-4049-a4f4-960b93bbca59)
Can you put the tooltip above? Same for loadout. otherwise the popup blocks the element below. I was frewuently getting blocked when moving the mouse from above, to get to the tabs below